index.js

```diff
@@ -226,9 +227,9 @@ const _isTenThousand = function (val) {
     isNotNegative(val) &&
     isPositive(val) &&
     !isNotPositive(val) &&
-    jQuery.equals(val, 10000) &&
-    isMatch(/10000/)(val) &&
-    val === 10000
+    jQuery.equals(val, TEN_THOUSAND) &&
+    isMatch(/TEN_THOUSAND/)(val) &&
```

Comment:
I just realised, since this is a regex it's now testing against the literal String `TEN_THOUSAND` instead of the number - you'll need to update it to something like this to use the new variable:

```suggestion
    isMatch(new RegExp(TEN_THOUSAND))(val) &&
```

Otherwise the smoke & full tests fail because all numbers are not `TEN_THOUSAND`, and this function doesn't test Strings.